components/cstudio-forms/controls/rte.js

```diff
@@ -546,7 +546,7 @@ YAHOO.extend(CStudioForms.Controls.RTE, CStudioForms.CStudioFormField, {
 						ed.contextControl._onChange(null, ed.contextControl);				
 			        });
 			        ed.onChange.add(function(ed, l) {
-			        	ed.contextControl.resizeEditor(ed);
+			        	//ed.contextControl.resizeEditor(ed);
```

Comment:
It's ok, This line was causing the issue, it was unnecessary. I test it in all browsers. 
I'm going to remove it.